pennylane_lightning/core/src/simulators/lightning_tensor/tncuda/TNCudaBase.hpp

```diff
@@ -519,14 +516,6 @@ class TNCudaBase : public TensornetBase<PrecisionT, Derived> {
 
         PL_CUDA_IS_SUCCESS(cudaStreamSynchronize(getDevTag().getStreamID()));
```

Comment:
Should we pass a null pointer for `stateNorm` argument in L514? `stateNorm2` is no longer used, and according to the [docs](https://docs.nvidia.com/cuda/cuquantum/latest/cutensornet/api/functions.html) `Providing a NULL pointer will ignore norm calculation. `